source/extensions/common/wasm/context.cc

```diff
@@ -543,6 +543,10 @@ Context::findValue(absl::string_view name, Protobuf::Arena* arena, bool last) co
     return CelValue::CreateStringView(root_id());
   case PropertyToken::PLUGIN_VM_ID:
     return CelValue::CreateStringView(wasm()->vm_id());
+  case PropertyToken::FILTER_STATE:
+    return Protobuf::Arena::Create<Filters::Common::Expr::FilterStateWrapper>(arena,
```

Comment:
I know nothing about CEL, but shouldn't this be wrapped in `CelValue::CreateMap()`?

Reply:
`Produce` returns a map.